com.github.wwmm.pulseeffects.yml

```diff
@@ -493,7 +491,7 @@ modules:
               - LIBDIR=lib
```

Comment:
I found the final zita-convolver issue. The update to 4.0.3 changed the behaviour of the LIBDIR variable to instead expect the absolute path (it now includes the PREFIX).